server/config/config.go

```diff
@@ -198,10 +198,11 @@ func Convert(cfg *Config) *config.Config {
 		Policy: "noop",
 	}
 	baseConfig.Services = map[string]config.Service{
-		"frontend": cfg.mustGetService(0),
-		"history":  cfg.mustGetService(1),
-		"matching": cfg.mustGetService(2),
-		"worker":   cfg.mustGetService(3),
+		"frontend":          cfg.mustGetService(0),
+		"history":           cfg.mustGetService(1),
+		"matching":          cfg.mustGetService(2),
+		"internal-frontend": cfg.mustGetService(3),
```

Comment:
Wasn't this removed in more recent PR? Do we need it here? cc: @dnr

Reply:
I'm not 100% sure what's going on here.. this is programatically building the static config for an in-memory/sqlite server? In that case, don't bother with internal-frontend, it's unnecessary (unless you're using an Authorizer, which you're not, right?)

Reply:
yea it's not used. However i get a server error if internal-frontend is not set

Reply:
```
...
received non-nil error from function "go.temporal.io/server/temporal".ServerOptionsProvider
	/home/player0/go/pkg/mod/go.temporal.io/server@v1.18.1-0.20230203202137-a4943bbd2f2f/temporal/fx.go:159:
config validation error: "internal-frontend" service is missing in config
```

Reply:
in `server/server.go`, change
```
		temporal.ForServices(temporal.Services),
```
to
```
		temporal.ForServices(temporal.DefaultServices),
```

Reply:
also, I'd drop the Authorizer and ClaimMapper stuff from that file. you shouldn't be using those in a local server

Reply:
ok thank you!

Reply:
what if we are running e2e tests and want to check the UI behavior when the authrorizer denies a request

Reply:
:shrug: I guess there's no harm, but it seems hard to do an effective test using only the builtin choices of authorizer+claimmapper. I would want to provide a dummy impl that does what I want, which means I'd have to rebuild the binary and override those bits anyway